dolphinscheduler-master/src/main/java/org/apache/dolphinscheduler/server/master/registry/ServerNodeManager.java

```diff
@@ -81,11 +82,29 @@ public class ServerNodeManager implements InitializingBean {
     private final ReentrantReadWriteLock.ReadLock workerNodeInfoReadLock = workerNodeInfoLock.readLock();
     private final ReentrantReadWriteLock.WriteLock workerNodeInfoWriteLock = workerNodeInfoLock.writeLock();
 
+    private final ReentrantReadWriteLock registryWorkerGroupLock = new ReentrantReadWriteLock();
+    private final ReentrantReadWriteLock.ReadLock registryWorkerGroupReadLock = registryWorkerGroupLock.readLock();
+    private final ReentrantReadWriteLock.WriteLock registryWorkerGroupWriteLock = registryWorkerGroupLock.writeLock();
+
+    private final ReentrantReadWriteLock dbWorkerGroupLock = new ReentrantReadWriteLock();
+    private final ReentrantReadWriteLock.ReadLock dbWorkerGroupReadLock = dbWorkerGroupLock.readLock();
+    private final ReentrantReadWriteLock.WriteLock dbWorkerGroupWriteLock = dbWorkerGroupLock.writeLock();
```

Comment:
I don' t think we need to add these locks and the `registryWorkerGroupNodes`, `dbWorkerGroupNodes` these two map.

Reply:
> I don' t think we need to add these locks and the `registryWorkerGroupNodes`, `dbWorkerGroupNodes` these two map.

yes, u are right, these locks are really unnecessary